NuKeeper/Commands/MultipleRepositoryCommand.cs

```diff
@@ -1,103 +1,104 @@
-
-
-        protected MultipleRepositoryCommand(IGitHubEngine engine, IConfigureLogger logger, IFileSettingsCache fileSettingsCache)
-            : base(engine, logger, fileSettingsCache)
-        {
-        }
-
-        protected override ValidationResult PopulateSettings(SettingsContainer settings)
-        {
-            var baseResult = base.PopulateSettings(settings);
-            if (!baseResult.IsSuccess)
-            {
-                return baseResult;
-            }
-
-            var regexIncludeReposValid = PopulateIncludeRepos(settings);
-            if (!regexIncludeReposValid.IsSuccess)
-            {
-                return regexIncludeReposValid;
-            }
-
-            var regexExcludeReposValid = PopulateExcludeRepos(settings);
-            if (!regexExcludeReposValid.IsSuccess)
-            {
-                return regexExcludeReposValid;
-            }
-
-            var fileSettings = FileSettingsCache.GetSettings();
-
-            const int defaultMaxReposChanged = 10;
-            settings.UserSettings.MaxRepositoriesChanged = Concat.FirstValue(
-                AllowedMaxRepositoriesChangedChange, fileSettings.MaxRepo, defaultMaxReposChanged);
-
-            return ValidationResult.Success;
-        }
-
-        private ValidationResult PopulateIncludeRepos(SettingsContainer settings)
-        {
-            var settingsFromFile = FileSettingsCache.GetSettings();
-            var value = Concat.FirstValue(IncludeRepos, settingsFromFile.IncludeRepos);
-
-            if (string.IsNullOrWhiteSpace(value))
-            {
-                settings.SourceControlServerSettings.IncludeRepos = null;
-                return ValidationResult.Success;
-            }
-
-            try
-            {
-                settings.SourceControlServerSettings.IncludeRepos = new Regex(value);
-            }
-            catch (Exception ex)
-            {
-                return ValidationResult.Failure($"Unable to parse regex '{value}' for IncludeRepos: {ex.Message}");
-            }
-
-            return ValidationResult.Success;
-        }
-
-        private ValidationResult PopulateExcludeRepos(SettingsContainer settings)
-        {
-            var settingsFromFile = FileSettingsCache.GetSettings();
-            var value = Concat.FirstValue(ExcludeRepos, settingsFromFile.ExcludeRepos);
-
-            if (string.IsNullOrWhiteSpace(value))
-            {
-                settings.SourceControlServerSettings.ExcludeRepos = null;
-                return ValidationResult.Success;
-            }
-
-            try
-            {
-                settings.SourceControlServerSettings.ExcludeRepos = new Regex(value);
-            }
-            catch (Exception ex)
-            {
-                return ValidationResult.Failure($"Unable to parse regex '{value}' for ExcludeRepos: {ex.Message}");
-            }
-
-            return ValidationResult.Success;
-        }
-    }
-}
+using System;
```

Comment:
Did anything happen here?

Reply:
A using statement was added is the only functional change

There was a space at the end of each line which Visual Studio wanted to remove. As above am happy to revert this if you like.

This occurred in 
NuKeeper.Tests\Configuration\FileSettingsReaderTests.cs
NuKeeper\Commands\MultipleRepositoryCommand.cs
NuKeeper.Abstractions\Configuration\FileSettings.cs

![image](https://user-images.githubusercontent.com/7226370/47704503-c27d0100-dc1b-11e8-9c95-2e6929bb100c.png)